frictionless/resources/table.py

```diff
@@ -331,19 +334,28 @@ def row_stream():
 
                 # Primary Key Error
                 if is_integrity and self.schema.primary_key:
-                    cells = tuple(row[name] for name in self.schema.primary_key)
-                    if set(cells) == {None}:
-                        note = 'cells composing the primary keys are all "None"'
-                        error = errors.PrimaryKeyError.from_row(row, note=note)
-                        row.errors.append(error)
+                    try:
+                        cells = self.primary_key_cells(row, self.dialect.header_case)
+                    except KeyError:
+                        # Row does not have primary_key as key
```

Comment:
```suggestion
                        # Row does not have primary_key as label
```

Plus aligné avec le vocabulaire utilisé ailleurs non ?